app/src/main/java/com/example/sabonit/Cart.java

```diff
@@ -85,4 +85,13 @@ public void setOrdersList(ArrayList<Order> ordersList) {
         this.ordersList = ordersList;
     }
 
+    /**
+     * check if the order list is empty or not
+     * @return empty - true, not empty - false
+     */
+    public boolean isCartEmpty()
+    {
+        return this.ordersList.size() == 0;
+    }
```

Comment:
OK, great function to use, simple and nice that you added the Javadoc to it

Reply:
thanks!